watchman/views.py

```diff
@@ -69,6 +69,13 @@ def status(request):
     return response, http_code, {WATCHMAN_VERSION_HEADER: __version__}
 
 
+@non_atomic_requests
```

Comment:
This is a django db transaction setting, probably not needed here?

Reply:
Hmm, yeah, not needed here. Updated to remove.